resources/measures/ResidentialConstructionsFacadesShared/tests/FacadesShared_Test.rb

```diff
@@ -16,6 +16,7 @@ def test_single_family_detached
   end
 
   def test_single_family_attached
+    skip
```

Comment:
Why is this test being skipped?

Reply:
@shorowit the shared facades measure is now irrelevant and has been removed